src/benchmarks/micro/corefx/System.Collections/Sort.cs

```diff
@@ -10,26 +10,49 @@
 
 namespace System.Collections
 {
+    // TODO: Where should the below types reside?
+
+    public readonly struct IntStruct : IComparable<IntStruct>
+    {
+        readonly int _value;
+
+        public IntStruct(int value) => _value = value;
+
+        public int CompareTo(IntStruct other) => _value.CompareTo(other._value);
+    }
+
+    public class IntClass : IComparable<IntClass>
+    {
+        readonly int _value;
+
+        public IntClass(int value) => _value = value;
+
+        public int CompareTo(IntClass other) => _value.CompareTo(other._value);
+    }
+
```

Comment:
It may be useful to also cover large struct with multiple fields.

Reply:
Yes absolutely, I will add later when we have uncovered the Span regressions :)

Reply:
@jkotas added `BigStruct`, does it look ok?